tensorflow-cnn/cnn_example_gpu.py

```diff
@@ -0,0 +1,162 @@
+# Experiment setup
+import argparse
+import datetime
+import numpy
+import time
+import scipy
+import sigopt
+import scipy.io
+import tensorflow as tf
+import math
+from skimage.color import rgb2gray
+from sklearn import preprocessing
+from sklearn.cross_validation import train_test_split
+
+conn = sigopt.Connection(client_token=client_token)
```

Comment:
Oh this was pilfered from https://github.com/sigopt/sigopt-examples/blob/3677e8bd6fc9d79fc561091f10796e98aaeb82ac/tensorflow-cnn/cnn_example_cpu.py#L76. But now that I look at that line it doesn't really make any sense. Shouldn't we just reuse that Connection object?